src/utils.py

```diff
@@ -0,0 +1,5 @@
+import logging
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+log.addHandler(logging.StreamHandler())
```

Comment:
I really recommend we switch from standard library logging to loguru: https://github.com/Delgan/loguru

It gives you dramatically better logs with no configuration. It also defaults to `INFO` level logs so you don't even need to set that.

<img width="679" alt="Screenshot 2024-05-13 at 11 25 54 PM" src="https://github.com/AmericanRedCross/street-view-green-view/assets/2721979/ac74251b-2710-479d-9238-de3c47d0749e">

<details>
<summary>Source code</summary>

```python
import logging

from loguru import logger

log = logging.getLogger(__name__)
log.setLevel(logging.INFO)
log.addHandler(logging.StreamHandler())

def do_the_thing():
    print("Here's standard library")
    log.info("This is an info message")
    print("---")
    print("Here's loguru")
    logger.info("This is an info message")
    logger.warning("Something is not right.")
    logger.success("I've completed the thing!")

if __name__ == "__main__":
    do_the_thing()
```

</details>

Reply:
Loguru is used in latest revision